msvc2015/Common/Common.vcxproj

```diff
@@ -134,13 +134,13 @@
     </None>
     <CustomBuild Include="..\..\util\Version.cpp.in">
       <FileType>Document</FileType>
-      <Command Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">"$(SolutionDir)..\python.exe" "$(SolutionDir)..\cmake\make_versioncpp.py" "$(SolutionDir).." "MSVC 2015"</Command>
-      <Command Condition="'$(Configuration)|$(Platform)'=='Release-XP|Win32'">"$(SolutionDir)..\python.exe" "$(SolutionDir)..\cmake\make_versioncpp.py" "$(SolutionDir).." "MSVC 2015"</Command>
+      <Command Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">"$(SolutionDir)..\python27.exe" "$(SolutionDir)..\cmake\make_versioncpp.py" "$(SolutionDir).." "MSVC 2015"</Command>
+      <Command Condition="'$(Configuration)|$(Platform)'=='Release-XP|Win32'">"$(SolutionDir)..\python27.exe" "$(SolutionDir)..\cmake\make_versioncpp.py" "$(SolutionDir).." "MSVC 2015"</Command>
```

Comment:
Must to be `python2.7.exe`

Reply:
Fixed.

Reply:
Why does it still depend on Python2?

Reply:
Well, make_versioncpp.py is still a Python 2 script.

Reply:
@geoffthemedio Switch to python 3 will be done in https://github.com/freeorion/freeorion/pull/2653

Reply:
So is the plan to include Python 2.7 in the next SDK for months / years after it's needed, or put out another SDK update soon after that just removes the Python2 stuff?

Reply:
@geoffthemedio At least python2 should be present in case of 0.4.9.1 release.